### PR TITLE
Add rlpipes

### PR DESCRIPTION
**Pull request by millerh1**
Submitting RLPipes, A standardized R-loop-mapping pipeline, to bioconda.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by millerh1** [comment]
@BiocondaBot please add label

File: recipes/rlpipes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - pysradb
+    - python
+  run:
+    - click
+    - pandas ==1.2.0
+    - pyfastx
+    - pysam
+    - snakemake-minimal ==6.4.0
+    - pysradb
+    - python
```
[the remark this answers is not in the document]
Comment (millerh1) [reply recipes/rlpipes/meta.yaml:30]: Hi Simon, there's an ongoing bug in > `6.4.0` snakemake that prevents parallel operations from functioning correctly when calling the API -- [issue](https://github.com/snakemake/snakemake/issues/1166). I'm watching the status of the bug and would be happy to push a completely relaxed version of this once it is fixed. For now, I've changed the requirement to:

```yaml
- snakemake-minimal >=5.20.1,=<6.4.0
```

And I tried to remove the pandas pin, but it seems to be pinned in `pysradb` and when I remove it on my recipe, it leads to a failure: 

```shell
raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (pandas 1.3.3 (/home/millerh1/miniconda3/conda-bld/rlpipes_1632407817263/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.9/site-packages), Requirement.parse('pandas==1.2.0'), {'rlpipes'})
Tests failed for rlpipes-0.9.0-py_0.tar.bz2 - moving package to /home/millerh1/miniconda3/conda-bld/broken
WARNING:conda_build.build:Tests failed for rlpipes-0.9.0-py_0.tar.bz2 - moving package to /home/millerh1/miniconda3/conda-bld/broken
WARNING conda_build.build:tests_failed(2955): Tests failed for rlpipes-0.9.0-py_0.tar.bz2 - moving package to /home/millerh1/miniconda3/conda-bld/broken
TESTS FAILED: rlpipes-0.9.0-py_0.tar.bz2
(base) millerh1@biolinx:~/projects/RLSuite/bioconda-rec
```
To relax this requirment us much as possible, I have changed to the following:

```yaml
- pandas <=1.2.0
```

This issue is also referenced [here](https://github.com/saketkc/pysradb/issues/141)

File: recipes/rlpipes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+test:
+  imports:
+    - rlpipes
+  commands:
+    - RLPipes --help
+
+about:
+  home: "https://github.com/Bishop-Laboratory/RLPipes"
+  license: MIT
+  license_family: MIT
```
[the remark this answers is not in the document]
Comment (millerh1) [reply recipes/rlpipes/meta.yaml:41]: Done :)

File: recipes/rlpipes/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (millerh1) [reply recipes/rlpipes/meta.yaml:30]: @simonbray Let me know if this is acceptable (I didn't want to mark resolve unless this is ok)

[the remark this answers is not in the document]
Comment (simonbray) [reply recipes/rlpipes/meta.yaml:29]: That's great, it looks like the pins are there for a good reason. Thanks a lot for making the changes!

**Review by simonbray — APPROVED** [review]

**Comment by millerh1** [comment]
@simonbray Thank you for the review! I am just a bit confused though on why the build seems to be failing now. `conda build` does fine locally and I want to try the `mulled-build` but  I am having a bit of trouble getting `circleci cli` to run properly on my machine (this is the issue I'm having). Do you have any suggestions?

**Comment by millerh1** [comment]
@simonbray I've reverted to the previous, but I actually forgot to add the license file. I'm going to try adding it now without changing anything else and see if it works ok.

**Comment by millerh1** [comment]
@simonbray It appears to have worked!! Please let me know if you need any additional changes for this to get merged!

**Comment by simonbray** [comment]
@bioconda-bot please merge

**Comment by simonbray** [comment]
Not sure how adding a license file can make a difference, but I went ahead and merged.

**Comment by millerh1** [comment]
Thank you :)

**Pull request merged**